stroeer/pages/section/v1/section_page.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package stroeer.pages.section.v1;
+
+import "stroeer/core/v1/article.proto";
+
+option go_package = "github.com/stroeer/tapir/go/stroeer/pages/section/v1;section";
+option java_multiple_files = true;
+option java_package = "de.stroeer.pages.section.v1";
+
+message SectionPage {
+  repeated Stage stages = 1;
```

Comment:
the page itself also needs configuration, doesn't it?

Reply:
could be added later